src/Image.cc

```diff
@@ -189,6 +190,25 @@ NAN_METHOD(Image::GetSource){
   info.GetReturnValue().Set(Nan::New<String>(img->filename ? img->filename : "").ToLocalChecked());
 }
 
+/*
+ * Get raw data.
+ */
+NAN_GETTER(Image::GetRawData) {
+  Image *img = Nan::ObjectWrap::Unwrap<Image>(info.This());
+  if (img->_surface) {
+    // Return raw ARGB data -- just a memcpy()
+    cairo_surface_t *surface = img->_surface;
+    cairo_surface_flush(surface);
+    const unsigned char *data = cairo_image_surface_get_data(surface);
+     unsigned int nBytes = img->width * img->height * 4;
+    Local<Object> buf = Nan::CopyBuffer(reinterpret_cast<const char*>(data), nBytes).ToLocalChecked();
+    info.GetReturnValue().Set(buf);
+  } else {
+    info.GetReturnValue().Set(Nan::Undefined());
+  }
+  return;
```

Comment:
No need to `return` at the end of the function ☺️

Reply:
Also don't need the `else { set undefined }` block -- that's implicit if you didn't set another return value.